src/Microsoft.ML.TimeSeries/SrCnnEntireAnomalyDetector.cs

```diff
@@ -880,18 +922,20 @@ private void CalculateExpectedValueByFft(double[] data)
                 FftUtils.ComputeBackwardFft(_fftRe, _fftIm, _ifftRe, _ifftIm, length);
             }
 
-            private void CalculateBoundaryUnit(double[] data, bool[] isAnomalys)
+            private void CalculateBoundaryUnit(double[] data, bool[] isAnomalies)
             {
                 int window = Math.Min(data.Length / 3, 512);
                 double trendFraction = 0.5;    // mix trend and average of trend
                 double trendSum = 0;
                 int calculationSize = 0;
+                bool closeToZero = true;
```

Comment:
nit, so by default `closeToZero` is true and `closeToZero &= trend < _eps` ? this seems a bit un-intuitive, you mean `closeToZero = false` and `closeToZero |= trend < _eps` or I misunderstood this variable?